mappings/net/minecraft/client/gui/screen/realms/RealmsMainScreen.mapping

```diff
@@ -59,60 +70,140 @@ CLASS net/minecraft/unmapped/C_uncfwjlr net/minecraft/client/gui/screen/realms/R
 		ARG 1 server
+	METHOD m_nzxelnxl getVersionText (Ljava/lang/String;I)Lnet/minecraft/unmapped/C_rdaqiwdt;
+		ARG 0 version
+		ARG 1 color
+	METHOD m_oqlgpiwx (Ljava/lang/Integer;)V
+		ARG 1 pendingInviteCount
+	METHOD m_orgobjsz (Ljava/lang/Boolean;)V
+		ARG 1 trialAvailable
+	METHOD m_oteeiouk refresh ()V
+	METHOD m_qaviylvw (Lnet/minecraft/unmapped/C_yyprjalz$C_zjxvgcbt;)V
+		ARG 1 serverList
 	METHOD m_qbrpgnib leaveServer (ZLnet/minecraft/unmapped/C_pwmwypfb;)V
 		ARG 1 confirmed
 		ARG 2 server
 	METHOD m_qhhosbao isSnapshot ()Z
 	METHOD m_qwnzuazr dismissNotification (Ljava/util/UUID;)V
 		ARG 1 notificationUuid
+	METHOD m_rozcbqng (Lnet/minecraft/unmapped/C_buwziidm;)V
+		ARG 1 button
 	METHOD m_rtkzebzs setupDataFetcher (Lnet/minecraft/unmapped/C_yyprjalz;)Lnet/minecraft/unmapped/C_tfgqmjhb$C_zolmsrgs;
 		ARG 1 dataFetcher
+	METHOD m_smqnphhm resetServerList ()V
+	METHOD m_sruatcui (Ljava/util/List;Lnet/minecraft/unmapped/C_ivysvyzf;)Ljava/lang/Object;
+		ARG 1 client
+	METHOD m_ssulbyvp createFooter (Lnet/minecraft/unmapped/C_uncfwjlr$C_ojhyljtj;)Lnet/minecraft/unmapped/C_xseakzsd;
+		ARG 1 loadStatus
 	METHOD m_suxyugra getSelectedServer ()Lnet/minecraft/unmapped/C_pwmwypfb;
+	METHOD m_tjyokdrg resetDataFetchTasks ()V
+	METHOD m_tmsuzsmk (Lnet/minecraft/unmapped/C_buwziidm;)V
+		ARG 1 button
+	METHOD m_tnttqbje (Ljava/util/List;)V
+		ARG 1 notifications
+	METHOD m_tsonseba (Lnet/minecraft/unmapped/C_aaohlqkp$C_pwjtucac;)V
+		ARG 1 error
+	METHOD m_ugllbcat (Lnet/minecraft/unmapped/C_pwmwypfb;Z)V
+		ARG 2 confirmed
 	METHOD m_uwkoxluc shouldLeaveButtonBeActive (Lnet/minecraft/unmapped/C_pwmwypfb;)Z
 		ARG 1 server
+	METHOD m_uxqnobsd (Lnet/minecraft/unmapped/C_yyprjalz;Lnet/minecraft/unmapped/C_gpedgsfu;)V
+		ARG 2 news
 	METHOD m_uyfoaanv configureClicked (Lnet/minecraft/unmapped/C_pwmwypfb;)V
 		ARG 1 serverData
 	METHOD m_veeacmfr isSelfOwnedServer (Lnet/minecraft/unmapped/C_pwmwypfb;)Z
 		ARG 1 serverData
-	METHOD m_yemkrboo (Lnet/minecraft/unmapped/C_pwmwypfb;Lnet/minecraft/unmapped/C_wrmtlwqx;Z)V
+	METHOD m_vjqotsrp (Lnet/minecraft/unmapped/C_buwziidm;)V
+		ARG 1 button
+	METHOD m_vpsrhejw updateLayout ()V
+	METHOD m_wzjaspte (Ljava/util/List;Ljava/lang/Object;)V
+		ARG 2 unused
+	METHOD m_xhxonkqe (Lnet/minecraft/unmapped/C_buwziidm;)V
+		ARG 1 button
+	METHOD m_xixvhmyu createLayout (Lnet/minecraft/unmapped/C_uncfwjlr$C_ojhyljtj;)Lnet/minecraft/unmapped/C_cqroimpa;
+		ARG 1 loadStatus
+	METHOD m_ybrbefcw (Ljava/util/UUID;Ljava/lang/Object;)V
+		ARG 2 unused
+	METHOD m_ydjnxzgt (Ljava/util/UUID;Lnet/minecraft/unmapped/C_auzifczy;)Z
+		ARG 1 notification
+	METHOD m_yemkrboo play (Lnet/minecraft/unmapped/C_pwmwypfb;Lnet/minecraft/unmapped/C_wrmtlwqx;Z)V
 		ARG 0 server
 		ARG 1 screen
+		ARG 2 allowSnapshot
+	METHOD m_yjygqnjl getVersionText (Ljava/lang/String;Z)Lnet/minecraft/unmapped/C_rdaqiwdt;
+		ARG 0 version
+		ARG 1 compatible
+	METHOD m_ymlzuivp getOwnedNonExpiredWorldIds ()Ljava/util/List;
```

Comment:
```suggestion
	METHOD m_ymlzuivp getActiveOwnedWorldIds ()Ljava/util/List;
```
`active` instead of `nonExpired`? not sure if it's worth the change, since it loses clarity for conciseness

Reply:
there's another method called something like `isOwnedNonExpired`, too, which isn't great but at least its clear